### PR TITLE
fix: references CurrentGraph

### PR DESCRIPTION
**Pull request by tanqiangyes**
The previous fixes prevented functions that referenced CurrentGraph functions from compiling, this commit modifies them and allows them to compile successfully.

### CODE REVIEW

File: compiler/preprocess.go [hunk 1]
```diff
@@ -91,13 +92,7 @@ func (p *Preprocess) checkCreateLabelStmt(stmt *ast.CreateLabelStmt) {
...
 
 	if !stmt.IfNotExists {
-		graphName := p.sc.CurrentGraph()
-		if graphName == "" {
-			p.err = ErrGraphNotChosen
-			return
-		}
-
-		graph := p.sc.Catalog().Graph(graphName)
+		graph := p.sc.CurrentGraph()
```
Comment (lonng) [inline compiler/preprocess.go:95]: You have changed the semantic here. An `ErrGraphNotChosen` are expected if users didn't choose graph via `USE <graphName>`

File: compiler/preprocess.go [hunk 2]
```diff
@@ -116,13 +111,7 @@ func (p *Preprocess) checkCreateIndexStmt(stmt *ast.CreateIndexStmt) {
 		return
 	}
 
-	graphName := p.sc.CurrentGraph()
```
Comment (lonng) [inline compiler/preprocess.go:119]: Ditto, please fix them.

**Comment by gozssky** [comment]
Hi @tanqiangyes, do you still work on this PR? I have met this compile error in my work and wait for this PR to merge. If you do not have time to fix the error, I am willing to help. Please let me know if you have time to continue working on this issue, or if you need any assistance.

Anyway, thanks for your contribution to this project!

**Comment by tanqiangyes** [comment]
> Hi @tanqiangyes, do you still work on this PR? I have met this compile error in my work and wait for this PR to merge. If you do not have time to fix the error, I am willing to help. Please let me know if you have time to continue working on this issue, or if you need any assistance.
> 
> Anyway, thanks for your contribution to this project!

i will fix as soon as fast.

**Comment by tanqiangyes** [comment]
@lonng pls review it. if something error,  @me , tks

**Review by gozssky — APPROVED** [review]

**Pull request merged**